knowledge_base/Contests.md

```diff
@@ -0,0 +1,48 @@
+# Contests
+
+Contests is a new feature, based on contest contracts, available for communities on Common. This document outlines the implementation of Contests on our platform.
+
+A contest contract is an on-chain, repeating game which receives content submissions and votes on submitted content. It then determines the winning content based on relative voting power applied.
+
+Common Contests are an application of contest contracts using Common content. Winners receive a token-denominated prize from a community-designated pool.
+
+## Basic Rules
+
+Each community may have at most one contest running. Contests continue to run even if the prize pool is empty, but no payments will be made to winners in such cases.
+
+Admins cannot interfere with the content submission and voting processes. To be able to change constructor variables, admins must deploy a new contest contract with their new desired variables.
+
+On-chain data submission will be conducted and sponsored by Commonwealth Labs. As of 240509, Common takes a 10% fee from contest distributions.
+
+Contests logic is managed by `Contest.sol`, and financial processes by the `FeeManager.sol` contract.
```

Comment:
There is no wrong information here but we can expand the details.

Reply:
Anything particular you have in mind? I guess I figured that engineers could learn the details by heading to the contract files, and mainly just wanted to point them in the right place. But perhaps there are specific macro-level dynamics worth flagging?